backend/src/main/java/com/redhat/cloud/notifications/routers/handlers/notification/NotificationResource.java

```diff
@@ -660,8 +660,8 @@ public void appendBehaviorGroupToEventType(
     ) {
         if (this.backendConfig.isKesselRelationsEnabled(getOrgId(securityContext))) {
             final UUID workspaceId = this.workspaceUtils.getDefaultWorkspaceId(getOrgId(securityContext));
-            this.kesselAuthorization.hasCreatePermissionOnResource(securityContext, WorkspacePermission.EVENT_TYPES_VIEW, ResourceType.WORKSPACE, workspaceId.toString());
-            this.kesselAuthorization.hasCreatePermissionOnResource(securityContext, WorkspacePermission.BEHAVIOR_GROUPS_EDIT, ResourceType.WORKSPACE, workspaceId.toString());
+            this.kesselAuthorization.hasUpdatePermissionOnResource(securityContext, WorkspacePermission.EVENT_TYPES_VIEW, ResourceType.WORKSPACE, workspaceId.toString());
```

Comment:
I agree with calling update for the `BEHAVIOR_GROUPS_EDIT` permission, but do we need to also CheckForUpdate for view? I suppose if we had an older zookie and just did a view check we may get allowed but then denied for editing.

Reply:
See my comment above for some more context, but even though it says `EVENT_TYPES_VIEW`, i.e. view, the overall operation it is trying to do is relate a behaviour group with an event type (with the kind of table you generally use for many-to-many mappings). So it's kind of an update for both sides, depending on how you look at it.

EDIT is kind of staying on the safe side. :)